pandas/tests/tools/test_to_timedelta.py

```diff
@@ -179,3 +179,10 @@ def test_to_timedelta_nullable_int64_dtype(self):
         result = to_timedelta(Series([1, None], dtype="Int64"), unit="days")
 
         tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize("func", ["Timedelta", "to_timedelta"])
```

Comment:
can u test at 1ns and 1us as well to make sure we r preserving

Reply:
Done